jira_scraper/core/scraper.py

```diff
@@ -169,3 +182,24 @@ def run(self, backup_path: str = "jira_all_bugs.pickle"):
         stats = self.db_manager.get_collection_stats(
             constants.COLLECTION_NAME)
         print(f"Number of records: {stats.points_count}")
+
+    def _dbg_print(self, df: pd.DataFrame) -> None:
+        """
+        Debug utility method to print detailed DataFrame contents.
+
+        The debug printing only happens when Python isn't running
+        in optimized mode (-O flag not used).
+
+        Args:
+            df: pandas DataFrame to print in detailed format
+        """
+        if __debug__:
+            pd.set_option('display.max_rows', None)
+            pd.set_option('display.max_columns', None)
+            pd.set_option('display.width', None)
+            pd.set_option('display.max_colwidth', None)
+            for index, row in df.iterrows():
+                print(f"Row {index}:")
+                for column, value in row.items():
+                    print(f"  {column}: {value}")
+                print("-" * 50)
```

Comment:
If the method isn't used, could we get rid of it as well?

Reply:
I left it in case we want to see all the tickets/comments etc fetched from Jira later. It will be just one liner add for debugging. But no problem to remove it either